recipe/meta.yaml

```diff
@@ -80,8 +85,11 @@ test:
     - tclsh hello.tcl
     - tclsh{{ maj_min }} hello.tcl                       # [unix]
     - tclsh{{ maj_min.replace(".", "") }} hello.tcl      # [win]
-    - wish hello.tcl
-    - wish{{ maj_min }} hello.tcl                        # [unix]
+    # wish requires X11:
+    # wish: error while loading shared libraries: libX11.so.6: cannot open shared object file: No such file or directory
+    # The time has come to figure out how to handle missing system dependencies. cc @msarahan
+    # - wish hello.tcl
+    # - wish{{ maj_min }} hello.tcl                      # [unix]
```

Comment:
Think we should be able to uncomment this either because X11 is in the `cdt` or because we include these in the Docker image.

Reply:
the cdt isn't available at test time here.

Reply:
Hmm...so we can't add it as a test requirement?